WordPress/Classes/Networking/PeopleRemote.swift

```diff
@@ -13,21 +13,35 @@ class PeopleRemote: ServiceRemoteWordPressComREST {
         case UnknownError
     }
 
+
+    /// Specifies the number of entities to be retrieved on each query.
+    ///
+    private let pageSize = 50
```

Comment:
No specific reason, but sounds like a good call. I'll lower it to 20, now that we've got paging!